Updated VM SLA URL

richeney opened the pull request:
Old version was hard coded to an obsolete page.  New one should redirect to the latest automatically.

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

richeney committed:
Updated VM SLA URL

Old version was hard coded to an obsolete page.  New one should redirect to the latest automatically.
